spec/support/request_spec_helper.rb

```diff
@@ -0,0 +1,5 @@
+module RequestSpecHelper
+  def json
```

Comment:
Looks good, but concerned the method name doesn't obviously describe what this does. It could be misinterpreted as serialising json from a hash, for example. `json_response` ?